tests/test_apdb.py

```diff
@@ -56,13 +56,17 @@ def test_load_sources(self):
         sources = self.apdb.load_sources(limit=2)
         self.assertEqual(len(sources), 2)
 
+        sources = self.apdb.load_sources(exclude_flagged=True)
+        self.assertEqual(len(sources), 11)
+
     def load_sources_for_object(self):
-        sources = self.apdb.load_sources_for_object(224948952930189335)
+        sources = self.apdb.load_sources_for_object(224948952930189342)
```

Comment:
I'm assuming you had to change these to get a non-excluded bitmask? Might note that in a comment.